test/ref/strtok.c

```diff
@@ -0,0 +1,43 @@
+// 2024-02-14 Sven Michael Klose <pixel@hugbox.org>
```

Comment:
While extra tests are always appreciated - please put this one into a separate PR